2.0/parser/filing_parser.py

```
@@ -0,0 +1,1799 @@
+            elif form_type in ['sc 13d', 'sc 13d/a']:
+                # Only want Item X part
+                processed_header = re.search("item\s+\d+", processed_header).group(0)
+            elif form_type in ['sc 13g', 'sc 13g/a']:
+                # We want Item X with optional '(y)' afterwards
+                processed_header = re.search(r'item\s+\d+(?:\([a-z]\))?', processed_header).group(0)
+            else:
+                logging.error(f'Unsupported form type {form_type} passed to tidy_sections_to_df_row, unsure how to process section {processed_header}. Expecting a filing of known tidy types: {self.tidy_filing_types}')
+        
+            if processed_header in [x.lower() for x in known_list]:
+                # Create row in DF with accession_number column from self.filing_info['accession_number'].iloc[0], section_name column with processed_header, section_meaning column with header_mappings, and section_text column with text
+                if processed_header in section_dict.keys():
```

Ruff (0.11.9)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)